source/common/tcp/original_conn_pool.cc

```diff
@@ -366,7 +366,6 @@ OriginalConnPoolImpl::ActiveConn::ActiveConn(OriginalConnPoolImpl& parent)
     : parent_(parent),
       connect_timer_(parent_.dispatcher_.createTimer([this]() -> void { onConnectTimeout(); })),
       remaining_requests_(parent_.host_->cluster().maxRequestsPerConnection()), timed_out_(false) {
-
```

Comment:
Revert; no need to touch this file for whitespace

Reply:
Reverted.